packages/frontend/src/features/scheduler/components/SchedulerForm.tsx

```diff
@@ -182,6 +211,8 @@ type Props = {
     onBack?: () => void;
     loading?: boolean;
     confirmText?: string;
+    isAlert?: boolean;
```

Comment:
isAlert might lead to confusion, I prefer `isThreshold` or even `isThresholdAlert`

Reply:
Ok, fair enough, I will change it to isThresholdAlert to be the most explicit.